src/state/api.js

```diff
@@ -18,9 +19,20 @@ export async function getPingData(appName, pingName) {
 }
 
 export async function getMetricData(appName, metricName) {
+  let updatedMetricName = metricName;
+
+  Object.keys(UBLOCK_ORIGIN_PRIVACY_FILTER).forEach((filter) => {
+    if (updatedMetricName.includes(filter)) {
```

Comment:
This check should not be needed -- if `updatedMetricName` does not contain `filter` then the replace will just return the original string.